adapters/api/get-downstream-assets.js

```diff
@@ -1,14 +1,15 @@
-import fetch from "node-fetch";
 import {
-  getConnectorImage,
+  ATLAN_API_TOKEN,
+  ATLAN_INSTANCE_URL,
+} from "../utils/get-environment-variables.js";
+import {
   getCertificationImage,
+  getConnectorImage,
   getImageURL,
 } from "../utils/index.js";
+
+import fetch from "node-fetch";
 import stringify from "json-stringify-safe";
-import {
-  ATLAN_INSTANCE_URL,
-  ATLAN_API_TOKEN,
-} from "../utils/get-environment-variables.js";
 
 const ASSETS_LIMIT = 100;
 
```

Comment:
We recently changed code to fetch downstream assets. Can u PTAL and add the required changes. It should be there in v1 tag. Its a minor fix. That will give u correct lineage in every case.

Reply:
Done